modules/tinymce/src/core/main/ts/selection/SetSelectionContent.ts

```diff
@@ -104,6 +105,7 @@ const cleanContent = (editor: Editor, args: SetSelectionContentArgs) => {
 };
 
 const setContent = (editor: Editor, content: string, args: Partial<SetSelectionContentArgs> = {}): void => {
+  Deprecations.logFeatureDeprecationWarning('selectionSetContent');
```

Comment:
You can't just put this here. We use the method internally; it will cause this message to appear in normal use of the editor.

Reply:
Maybe separate this into two functions `setContentInternal` and `setContentExternal` where external is the one we expose in public APIs and add the message to and it just calls `setContentInternal`.